packages/flutter_test/lib/src/binding.dart

```diff
@@ -566,6 +560,26 @@ abstract class TestWidgetsFlutterBinding extends BindingBase
     }());
   }
 
+  void _verifyReportTestExceptionUnset(TestExceptionReporter valueBeforeTest) {
+    assert(() {
+      if (reportTestException != valueBeforeTest) {
+        // We can't report this error to their modified reporter because we
+        // can't be guaranteed that their reporter will cause the test to fail.
+        // So we reset the error reporter to its initial value and then report
```

Comment:
It's possible for the config script to change this to some other value that they want us to use in general. It might not be _defaultTextExceptionReporter.